soccer/src/soccer/planning/planner/line_kick_path_planner.hpp

```diff
@@ -47,6 +48,8 @@ class LineKickPathPlanner : public PathPlanner {
 
     PathTargetPathPlanner path_target_{};
     CollectPathPlanner collect_planner_{};
+    PivotPathPlanner pivot_planner_{};
```

Comment:
Looks like including pivoting in `line_kick` was started but never completed. @jacksherling Should we uncommit this? Or is this going to do something?

Reply:
Good catch, up to you. If we want line kick to handle rotation, maybe keep it for now. But if we're splitting up line kicker anyway we should uncommit.

Reply:
alright i'll uncommit and we can re-add later. there's a ticket to fix line kick for the millionth time